dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/insertmanager/insertionmodifier/IOSLegacyProblemRPLT144Modifier.java

```diff
@@ -0,0 +1,31 @@
+package org.dpppt.backend.sdk.ws.insertmanager.insertionmodifier;
+
+import java.util.List;
+import org.dpppt.backend.sdk.model.gaen.GaenKey;
+import org.dpppt.backend.sdk.semver.Version;
+import org.dpppt.backend.sdk.utils.UTCInstant;
+import org.dpppt.backend.sdk.ws.insertmanager.OSType;
+
+/**
+ * This key modifier makes sure, that rolling period is always set to 144. Default value according
+ * to EN is 144, so just set it to that. This allows to check for the Google-TEKs also on iOS.
+ * Because the Rolling Proximity Identifier is based on the TEK and the unix epoch, this should
+ * work. The only downside is that iOS will not be able to optimize verification of the TEKs,
+ * because it will have to consider each TEK for a whole day.
```

Comment:
@ineiti I agree with doing this modification on the download side. Hence my recommendation to at least specify the "cost" of doing this here for all devices.

Reply:
The longer we wait, this filter will no be needed in production anyway. Also, it can be configured via application properties. Default, this modifer is not used. So yes, I think specifing the cost is sufficent.